README.md

```diff
@@ -884,6 +884,17 @@ The region is defined as a rectangle in a routing layer.
 Example: `set_global_routing_region_adjustment {1.5 2 20 30.5}
                                                -layer Metal4 -adjustment 0.7`
 
+```
+set_global_routing_random [-seed seed] \
+                          [-capacities_perturbation_percentage percent] \
+                          [-perturbation_amount value]
+```
+The `set_global_routing_random` command enables random global routing results. The random global routing shuffles the order
+of the nets and randomly subtracts or add the capacities of a random set of edges.
+The `-seed` option sets the random seed, the `-capacities_perturbation_percentage` option sets the percentage of edges to
+perturb the capacities, and the `-perturbation_amount` option sets the perturbation value of the edge capacities.
```

Comment:
Do you have to use -capacities_perturbation_percentage and -perturbation_amount together?  I'm not sure what it would mean to use just one.